
Loading datasets, it could take some time....
Traceback (most recent call last):
  File "/dati/home/francesco.pugnaloni/TableEmbeddingsWithGNNs/experiments.py", line 21, in <module>
    run_GNNTE_experiment(**l[i])
  File "/dati/home/francesco.pugnaloni/TableEmbeddingsWithGNNs/GNNTE.py", line 456, in run_GNNTE_experiment
    train_test_pipeline(triples_path, graphs_path, checkpoint, out_channels, n_layers,
  File "/dati/home/francesco.pugnaloni/TableEmbeddingsWithGNNs/GNNTE.py", line 176, in train_test_pipeline
    all_data = load_test_training_stuff(triple_file, graph_file)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/dati/home/francesco.pugnaloni/TableEmbeddingsWithGNNs/GNNTE.py", line 143, in load_test_training_stuff
    gd = pickle.load(f)
         ^^^^^^^^^^^^^^
  File "/home/francesco.pugnaloni/anaconda3/envs/HPIPyG/lib/python3.11/site-packages/torch/storage.py", line 241, in _load_from_bytes
    return torch.load(io.BytesIO(b))
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/francesco.pugnaloni/anaconda3/envs/HPIPyG/lib/python3.11/site-packages/torch/serialization.py", line 815, in load
    return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/francesco.pugnaloni/anaconda3/envs/HPIPyG/lib/python3.11/site-packages/torch/serialization.py", line 1051, in _legacy_load
    typed_storage._untyped_storage._set_from_file(
KeyboardInterrupt